src/components/EventForm/EventForm.js

```diff
@@ -1,22 +1,18 @@
 import './EventForm.css';
 import { useState, useEffect } from 'react';
-import { courses, players } from '../../APICalls/sampleData';
 
-function EventForm() {
-  const currentUser = players.find((player) => player.id === 2); // need to make this dynamic to match what user is logged in
-  const userFriends = currentUser.friends.map((friendId) => {
-    return players.find((player) => player.id === friendId).name;
-  });
-  const [date, setDate] = useState('');
+function EventForm({courses, friends}) {
+  const today = new Date().toISOString().slice(0, 10);
+  const tomorrow = new Date(new Date(today).getTime() + 86400000).toISOString().slice(0, 10);
```

Comment:
I installed dayjs for what I'm currently working on, so ya'll are welcome to use it too as it makes this process a bit easier.